This patch is no longer needed since Lustre 2.2.  It is kept here to
allow older versions of Lustre to build against the same kernel as
Lustre 2.2 in order to allow upgrade of just Lustre without also
replacing the kernel at the same time.

Version 47: allow the ext3_map_inode_page() created param to be NULL
Version 46: Separate ->fs_private in struct file
Version 45: more robust and general dev_read_only for failover (b=4834)  
Version 44: fix link_path_walk_it() oops creating .foo in deleted "." (b=5548)
Version 43: fix remove_suid to not crash 2.6, and do anything on 2.4 (b=5695)
Version 42: export show_task()
Version 41: revalidate should check working dir is a directory (b=4134)
Version 40: >32000 subdirectories support for ext3 (b=3244)
Version 39: add EXPORT_SYMBOL(smp_num_siblings) to vanilla-2.4.24 (b=3966)
            eebperf (unfragmented 1MB write/read support)
            48-bit physical block support for extents
Version 38: drop dentry ref in ext3_add_link from open_connect_dentry (b=3266)
Version 37: fix htree rename-within-same-dir (b=3417), endianness (b=2447)
Version 36: don't dput dentry after error (b=2350), zero page->private (3119)
Version 35: pass intent to real_lookup after revalidate failure (b=3285)
Version 34: ext3 iopen assertion (b=2517), operations on deleted "." (b=2399)

 include/linux/lustre_version.h |    1 +
 1 files changed, 1 insertion(+)

--- /dev/null	Fri Aug 30 17:31:37 2002
+++ linux-2.4.18-18.8.0-l12-braam/include/linux/lustre_version.h	Thu Feb 13 07:58:33 2003
@@ -0,0 +1 @@
+#define LUSTRE_KERNEL_VERSION 47

_
